src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -22,7 +23,7 @@ public UserController(UserService userService, UserMapper userMapper) {
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto getByEmail(@RequestParam String email) {
+    public UserResponseDto getByEmail(@RequestParam @Valid String email) {
```

Comment:
How do you validate email here?
It is better to use custom email validation here.